cloudferry/lib/os/compute/nova_compute.py

```diff
@@ -732,17 +731,19 @@ def get_instances_list(self, detailed=True, search_opts=None, marker=None,
 
         active_computes = self.get_compute_hosts()
         active_servers = []
-
         for server in servers:
+            msg = "Instance '{0}' has been excluded from VMs list, " \
+                  "because %s".format(server.id)
+            if server.status not in ALLOWED_VM_STATUSES:
+                LOG.debug(msg, "the status '{0}' is not "
+                               "allowed.".format(server.status))
```

Comment:
Why not just write

    LOG.debug("Instance '%s' has been excluded from VMs list, because the "
              "status '%s' is not allowed.", server.id, server.status)